frontends/web/src/locales/en/app.json

```diff
@@ -372,13 +373,12 @@
           },
           "title": "Payment methods and fees"
         },
-        "privacyPolicy": "MoonPay privacy policy",
         "protection": {
-          "description": "The BitBoxApp does not collect any data when buying {{name}}, the incoming funds are treated like a regular transaction. MoonPay needs to collect some personal data to operate. Their Privacy Policy explains in detail how that data is handled.",
+          "description": "The BitBoxApp does not collect any data when buying {{name}}, the incoming funds are treated like a regular transaction. However MoonPay and Pocket need to collect some information to operate. Please refer to their respective privacy policies to see in more detail how the data is handled.",
```

Comment:
I think I am missing context, could you update the commit message with how this should work?

Is the following correct?

each exchange/broker terms has its own text, followed by a shared text (this one). which is also shared in the guide.

I am confused by that in the "own" text it's the exchange/broker name, but in the next paragraph it suddently mentioned other service that I didn't select. This would be less confusing if we just used generic term instead of the actual name.

Also what if we add 5 more exchanges broker one day, maybe the user doesn't even know about them if they are not available in different regions. Then the text lists them all?

ping @jadzeidan 

> However MoonPay, Pocket, Broker2, Exchange3, BrokerNotInMyRegion4 or P2PXchange need to collect..